BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+## @ GenerateKeys.py
+# Generate RSA Keys
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+##
+# Import Modules
+#
+import os
+import sys
+import re
+import glob
+import struct
+import shutil
+import subprocess
+import datetime
+import zipfile

```

Comment:
do we need to import re, glob, struct, zipfile, shutil ?

Reply:
Not required. I will remove them.